examples/systemcalls/systemcalls.c

```diff
@@ -17,6 +23,10 @@ bool do_system(const char *cmd)
  *   or false() if it returned a failure
 */
 
+    int ret = system(cmd);
+    if (ret == -1) {
+        return false;
+    }
```

Comment:
@vido2373 consider what happens when the command invoked by `system()` returns an error exit code which isn't -1.  Will you meet the requirements in the function definition block?